content/_includes/post.macros.njk

```diff
@@ -262,41 +262,6 @@ params:
 {% endmacro %}
 
 
-{# @docs
```

Comment:
@jgerigmeyer I take it we are not using this anywhere (anymore at least)?

Reply:
Yeah, I couldn't find any uses of it, and the build didn't error when I removed it. 🤷